frontend/lib/src/components/transaction_component/transaction_component.dart

```diff
@@ -74,10 +130,9 @@ class TransactionComponent extends StatelessWidget {
                 height: 13,
               ),
               if (type == 'TRANSFER') ...[
-                InputField(
-                  name: 'email',
-                  label: 'Email',
-                  validator: FormBuilderValidators.email(),
+                const InputField(
```

Comment:
don't make changes for transfer for now since it is for other task.

Reply:
Reverted